packages/node/src/login/oidc/oidcHandlers/RefreshTokenOidcHandler.spec.ts

```diff
@@ -246,4 +251,64 @@ describe("RefreshTokenOidcHandler", () => {
     );
     expect(result).not.toBeUndefined();
   });
+
+  it("throws if the IdP does not return an ID token", async () => {
+    // This builds the fetch function holding the refresh token...
+    const refreshTokenOidcHandler = new RefreshTokenOidcHandler(
+      mockTokenRefresher({
+        access_token: "some access token",
+        expired: () => false,
+        claims: () => (null as unknown) as IdTokenClaims,
+      }),
+      mockStorageUtility({})
+    );
+    const result = refreshTokenOidcHandler.handle(
+      mockOidcOptions({
+        refreshToken: "some refresh token",
+        client: {
+          clientId: "some client id",
+          clientSecret: "some client secret",
+        },
+      })
+    );
+    await expect(result).rejects.toThrow(
+      "The Identity Provider did not return an ID token on refresh, which prevents from getting the user's WebID."
```

Comment:
Shouldn't the IdP be referenced in this error message, e.g.:
```suggestion
      "The Identity Provider [${identityProvider}] did not return an ID token on refresh, which prevents us from getting the user's WebID."
```
(NOTE: I also added the word 'us' in that message to make is more readable)